docs/shop4cfcontext.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    
+    "isDefinedBy": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/isDefinedBy",
+      "@type": "ngsi-ld:Relationship"
+    },
+    
+    "involves": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/involves",
+      "@type": "ngsi-ld:Property"
+    },
+    
+    "happensAt": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/happensAt",
+      "@type": "ngsi-ld:Property"
+    },
+    
+    "workParameters": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/workParameters",
+      "@type": "TO BE DEFINED"
+    },
+    
+    "outputParameters": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/outputParameters",
+      "@type": "TO BE DEFINED"
+    }
+  }
```

Comment:
We should also add `status` attribute, right?

Reply:
Yes, but I was not sure. In theory if we use the context h`ttps://smartdatamodels.org/context.jsonld` there is already status specified. Is that kind of status good for us?

Reply:
I do not know, as I cannot find any specific definition of 'their' status attribute. 

Ours is defined [here in the botton](https://github.com/SHOP4CF/data-models/blob/master/docs/task.md). We defined specific dictionary of allowed values (e.g. pending, assigned, etc.).

For now, it seems we should leave this issue open (and thus not yet add `status` to this `.jsonld`) to be clarified later.

Reply:
![Status for Fiware](https://user-images.githubusercontent.com/26014548/123773422-f200d180-d8cc-11eb-8048-7a09b83a813c.png)

You are right the status defined in fiware is not well defined. I would say that we need to create our own Status attributue. Could you please specify which is the dictionary for the values? Do you confirm it is the one linked [above](https://github.com/SHOP4CF/data-models/blob/master/docs/task.md)?

Reply:
Yes, this is the outcome of my discussion with Kostas:

> `status` represents the current execution status with the following allowed values: `pending`, `assigned`, `inProgress`, `completed`, `paused`, `suspended`, `failed`